_posts/2020-08-01-Fortran-Newsletter-August-2020.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Fortran newsletter: August 2020"
+category: newsletter
+author: Ondřej Čertík, Milan Curcic, Laurence Kedward, and Jérémie Vandenplas
+---
+
+Welcome to the August 2020 edition of the monthly Fortran newsletter.
+The newsletter comes out on the first calendar day of every month
+and details Fortran news from the previous month.
+
+<ul id="page-nav"></ul>
+
+## fortran-lang.org
+
+Work has continued on the Fortran-lang website, including a new community page and additional tutorial content:
+
+* [#120](https://github.com/fortran-lang/fortran-lang.org/pull/120): updated the
+  [Book section](https://fortran-lang.org/learn/) with a comprehensive list of
+  books about Fortran
+
+Ongoing work:
+
+* [#117](https://github.com/fortran-lang/fortran-lang.org/issues/117): Adding a
+  Benchmarks section, a new dedicated repository was created at
+  https://github.com/fortran-lang/benchmarks and many details have been
+  discussed in [issues](https://github.com/fortran-lang/benchmarks/issues) there
+
+
+
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+
+## Fortran Standard Library
+
+What's new in the Fortran Standard Library:
+
+
+Some miscellaneous fixes and improvements:
+
```

Comment:
```suggestion
Main ongoing discussions:
* [#225](https://github.com/fortran-lang/stdlib/issues/225): Name convention for derived types in`1stdlib`
* [#224](https://github.com/fortran-lang/stdlib/issues/224): Handling and propagating errors inside `stdlib`
* [#221](https://github.com/fortran-lang/stdlib/issues/221): APY for a bitset data type
* [#201](https://github.com/fortran-lang/stdlib/issues/201): API for file system operations
```

Reply:
in`1stdlib` -> in `stdlib`

Reply:
> in`1stdlib` -> in `stdlib`

Changed!

Reply:
Are these info suited for the newsletter? It not, it can be resolved.

Reply:
Also put space between in`stdlib`. 

It doesn't allow me to commit it for some reason --- but simply push the change in if you can.